tests/features/solution/homepage_related_content.feature

```diff
@@ -35,6 +35,7 @@ Feature: Solution homepage
       | title               | document type | short title | created           | body                    | spatial coverage | policy domain | solution                     | state     |
       | IS protocol draft 2 | Document      | IS draft 2  | 2018-10-04 8:08am | Next proposition draft. | European Union   | E-inclusion   | Information sharing protocol | validated |
 
+  @clearStaticCache
```

Comment:
This actually causes the test to create a false positive.
I tested this manually, I already had the data in MATOMO and I only did the following:
* Reset on the current branch.
* Restore the database and run the updates.
* Visit `/solution/eli-annotation-tool/distribution/eli-annotation-tool` which is the distribution (and solution) I am testing. The header is cached.
* Run `./vendor/bin/drush sql-query "delete from queue where data not like '%2561%';" ` to clear out all items from the queue except from the one of the above distribution.
* Run the cron job.
* Through XDEBUG verify that the counters are updated (from 0 to 2) and the correct cache tag is invalidated.
* Re visit the distribution or solution page.
* The counter is not updated (not shown).

Clearing the cache shows the proper counter.

Reply:
As discussed with Claudiu, this does not really cause a false positive. The source of the issue is when the solution does not have a counter initially and then it retrieves one. This is caused because the downloads element had `'#access' => false` so it was not included in the render array and the cache tags were not added in the screen. Added a commit to always show even a hidden array so that cache can be invalidated!